kover-gradle-plugin/src/main/kotlin/kotlinx/kover/gradle/plugin/dsl/KoverNames.kt

```diff
@@ -0,0 +1,124 @@
+package kotlinx.kover.gradle.plugin.dsl
+
+import kotlinx.kover.gradle.plugin.commons.htmlReportTaskName
+import kotlinx.kover.gradle.plugin.commons.verifyTaskName
+import kotlinx.kover.gradle.plugin.commons.xmlReportTaskName
+import org.gradle.api.plugins.ExtensionContainer
+import org.gradle.api.tasks.TaskContainer
+
+public object KoverNames {
+    /**
+     * Name of the configurations to add dependency on Kover reports from another project.
+     */
+    public const val DEPENDENCY_CONFIGURATION_NAME = "kover"
+
+    /**
+     * Name of the project extension to configure Kover measurements.
+     */
+    public const val PROJECT_EXTENSION_NAME = "kover"
+
+    /**
+     * Name of the project extension to configure Kover reports.
+     */
+    public const val REPORT_EXTENSION_NAME = "koverReport"
+
+    /**
+     * Name of the XML report generation task for Kotlin JVM and Kotlin multiplatform projects.
+     */
+    public const val DEFAULT_XML_REPORT_NAME = "koverXmlReport"
+
+    /**
+     * Name of the HTML report generation task for Kotlin JVM and Kotlin multiplatform projects.
+     */
+    public const val DEFAULT_HTML_REPORT_NAME = "koverHtmlReport"
+
+    /**
+     * Name of the verification task for Kotlin JVM and Kotlin multiplatform projects.
+     */
+    public const val DEFAULT_VERIFY_REPORT_NAME = "koverVerify"
+
+    /**
+     * Name of the XML report generation task for [buildVariant] Android build variant for Android projects.
+     */
+    public fun androidXmlReport(buildVariant: String): String {
+        return xmlReportTaskName(buildVariant)
+    }
+
+    /**
+     * Name of the HTML report generation task for [buildVariant] Android build variant for Android projects.
+     */
+    public fun androidHtmlReport(buildVariant: String): String {
+        return htmlReportTaskName(buildVariant)
+    }
+
+    /**
+     * Name of the verification task for [buildVariant] Android build variant for Android projects.
+     */
+    public fun androidVerify(buildVariant: String): String {
+        return verifyTaskName(buildVariant)
+    }
+}
+
+/**
+ * Name of the XML report generation task for Kotlin JVM and Kotlin multiplatform projects.
+ *
+ * Same as a [KoverNames.DEFAULT_XML_REPORT_NAME].
```

Comment:
If they're the same, why `object KoverNames` should be exposed at all?

Reply:
Then we will need `koverXmlReportName` to call, we will need instance of `TaskContainer`, but it may not be available in some places.

Reply:
Why extension on TaskContainer is needed then, if receiver is not used?

Reply:
For a more convenient search constants without studying the documentation. 
There is no need to know about the existence of the `KoverNames` object, just type `tasks.kover` and the IDE will offer these constants